src/resources/2024/BatWings.ts

```diff
@@ -38,3 +39,19 @@ export function cauldronsRemaining(): number {
 export function flapChance(flaps = get("_batWingsFreeFights")): number {
   return flaps < 5 ? 1 / (2 + flaps) : 0;
 }
+
+/**
+ * Attempts to do bridge skip
+ * @returns whether we were successfully able to skip the bridge
+ */
+export function bridgeSkip(): boolean {
```

Comment:
You need wings equipped in order to make the skip available. Should libram handle that too or do we want callers to handle that?